rclcpp/include/rclcpp/time.hpp

```diff
@@ -189,7 +189,7 @@ class Time
    */
   RCLCPP_PUBLIC
   static Time
-  max();
+  max(rcl_clock_type_t clock_type = RCL_SYSTEM_TIME); // NOLINT
```

Comment:
```suggestion
  max(rcl_clock_type_t clock_type = RCL_SYSTEM_TIME);  // NOLINT
```

(otherwise I think uncrustify is going to complain)

Reply:
addressed with https://github.com/ros2/rclcpp/pull/2352/commits/66bd2130efa9391d6d5e54fba3bc23bcdb5413bc